### ISSUE TITLE
Render layout

### ISSUE THREAD

**Issue by ftomza** [qor/render#2]
Hi!

I render some template using layout (application.tmpl). But for some reason layout contains error. In this case I can't view layout error neither in IDE console nor in browser. How can I view such kind of error ?

**Comment by cryptix** [comment]
I ran into similar problems... the [render](https://github.com/qor/render/blob/9bc9220a811df16c414ede83b3c8d6aa65788e95/template.go#L46) function code isn't pretty, it ignores errors in a lot of cases.

**Comment by jinzhu** [comment]
@ftomza @cryptix I think errors will be printed to console at least? 

https://github.com/qor/render/blob/9bc9220a811df16c414ede83b3c8d6aa65788e95/template.go#L83
https://github.com/qor/render/blob/9bc9220a811df16c414ede83b3c8d6aa65788e95/template.go#L130

**Comment by cryptix** [comment]
No, they were not somehow and I got tired unwinding this _clever_ style, it's really tiresome to follow.

I posted my version of `Render()` [here](https://gist.github.com/cryptix/b69835719da7167a6183026e4f895c91).

**Comment by cryptix** [comment]
I started a [fork](https://github.com/qor/render/compare/master...cryptix:master) of this package with that patch and would love to see this merged.

I found that qor/admin also has code like this [here](https://github.com/qor/admin/blob/master/context.go#L209). It should also be improved but I'm unwilling to maintain a fork at this point as it is too large of an package to maintain a fork just for that size of change.
